docs/quickstarts/aws.md

```diff
@@ -0,0 +1,207 @@
+grafana_admin_password = "password-for-grafana"
+```
+
+For advanced cluster configurations and more information refer to the [AWS configuration
+guide.](../configuration-guides/platforms/aws.md)
+
+### Step 5: Create Lokomotive cluster
+
+Run the following command to create the cluster:
+
+```console
+lokoctl cluster install
+```
+Once the command finishes, your Lokomotive cluster details are stored in the path you've specified
+under `asset_dir`.
+
+## Verification
+
+A successful installation results in the output:
+
+```console
+module.aws-myawscluster.null_resource.bootkube-start: Still creating... [1m50s elapsed]
+module.aws-myawscluster.null_resource.bootkube-start: Still creating... [2m0s elapsed]
+module.aws-myawscluster.null_resource.bootkube-start: Creation complete after 2m6s [id=5156996152315868880]
+
+Apply complete! Resources: 118 added, 0 changed, 0 destroyed.
+
+Your configurations are stored in /home/imran/lokoctl-assets/myawscluster
+
+Now checking health and readiness of the cluster nodes ...
+
+Node              Ready    Reason          Message
+
+ip-10-0-39-75     True     KubeletReady    kubelet is posting ready status
+ip-10-0-39-78     True     KubeletReady    kubelet is posting ready status
+ip-10-0-39-29     True     KubeletReady    kubelet is posting ready status
+ip-10-0-12-241    True     KubeletReady    kubelet is posting ready status
+ip-10-0-12-244    True     KubeletReady    kubelet is posting ready status
+ip-10-0-12-249    True     KubeletReady    kubelet is posting ready status
+
+Success - cluster is healthy and nodes are ready!
+```
+
+Use the generated `kubeconfig` file to access the Kubernetes cluster and list nodes.
+
+```console
+export KUBECONFIG=./lokomotive-assets/auth/kubeconfig
+kubectl get nodes
+```
+
+## Cleanup
+
+To destroy the Lokomotive cluster, execute the following commmand:
+
+```console
+lokoctl cluster destroy --confirm
+```
+
+You can safely delete the working directory created for this quickstart guide if you no longer
+require it.
+
+## Troubleshooting
+
+### Stuck At Copy Controller Secrets
+
+If there is an execution error or no progress beyond the output provided below:
+
+```console
+...
+module.aws-myawscluster.null_resource.copy-controller-secrets: Still creating... (8m30s elapsed)
+module.aws-myawscluster.null_resource.copy-controller-secrets: Still creating... (8m40s elapsed)
+...
+```
+
+The error is because the `ssh_pubkeys` provided in the configuration is missing in the `ssh-agent`
+
+To rectify the error, you need to:
+
+1. Execute [Cleanup](#cleanup) step.
+2. Follow the steps [to add the SSH key to the
+   ssh-agent.](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent)
+3. Retry [Step 5.](#step-5-create-lokomotive-cluster)
+
+### IAM Permission Issues
+
+  * If the failure is due to insufficient permissions, check the IAM policy.
```

Comment:
Maybe we can remove this part? It doesn't seem too comprehensive.

Reply:
I think it makes sense if the user is using aws credentials that do not have appropriate access.

Reply:
I would elaborate it more then. Or point to some AWS docs with instruction for further debugging.